src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -34,20 +37,24 @@
 	</div>
 
 	<div
-		class="item flex flex-row items-center justify-end gap-2 text-sm lg:max-w-48 xl:max-w-none"
+		class="item flex flex-row items-center justify-end gap-2 text-sm transition-all duration-200 ease-in-out lg:max-w-48 xl:max-w-none"
 		class:sm:max-w-none={$authNotSignedIn}
+		class:sm:invisible={$authSignedIn}
+		class:1.5md:visible={$authSignedIn}
+		class:translate-x-full={$authSignedIn}
+		class:1.5md:translate-x-0={$authSignedIn}
 	>
-		<IconDfinity />
-		<span>
-			{replaceOisyPlaceholders($i18n.footer.text.developed_with)}
-			<ExternalLink
-				href="https://dfinity.org"
-				ariaLabel={replaceOisyPlaceholders($i18n.footer.alt.dfinity)}
-				color="blue"
-				iconVisible={false}
-			>
-				{replaceOisyPlaceholders($i18n.footer.text.dfinity)}
-			</ExternalLink>
-		</span>
+		<ExternalLink
+			href="https://dfinity.org"
+			ariaLabel={replaceOisyPlaceholders($i18n.footer.alt.dfinity)}
+			iconVisible={false}
+		>
+			<div class="flex flex-row items-center gap-2">
+				<IconDfinity />
```

Comment:
I missed that too, did I review the previous PR 😅. It's `IconDfinity` not `IconInternetComputer` or something?

Reply:
eheh same as above... i will re-check

Reply:
Just as follow up: I checked with both PM and Design team and that is indeed on purpose all DFINITY related.